<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="VijayaSachinLaptop" name="com.qa.crm.testcases.DealsTestCase" tests="2" failures="0" timestamp="10 Apr 2020 18:12:58 GMT" time="16.184" errors="1">
  <testcase name="user_moves_to_new_deals_page" time="16.184" classname="com.qa.crm.testcases.DealsTestCase"/>
  <testcase name="user_enters_deals_details" time="0.000" classname="com.qa.crm.testcases.DealsTestCase">
    <error type="org.testng.TestNGException" message="
Method user_enters_deals_details requires 1 parameters but 0 were supplied in the @Test annotation.">
      <![CDATA[org.testng.TestNGException: 
Method user_enters_deals_details requires 1 parameters but 0 were supplied in the @Test annotation.
	at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:191)
	at org.testng.internal.Parameters.createParameters(Parameters.java:127)
	at org.testng.internal.Parameters.createParameters(Parameters.java:376)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:451)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1270)
	at org.testng.internal.Invoker.createParameters(Invoker.java:985)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1075)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- user_enters_deals_details -->
</testsuite> <!-- com.qa.crm.testcases.DealsTestCase -->
